hugo/content/playground/_index.html

```diff
@@ -28,23 +28,67 @@
   MonacoEditorLanguageClientWrapper.addCodiconTtf();
 
   addEventListener('load', function() {
+    let getState = null; //: () => PlaygroundParameters
+
     const divRoot = document.getElementById("monaco-editor-root");
-    const style = "width:50%; display:inline-block; height: 100%";
-    const leftEditor = document.createElement("div");
-    leftEditor.setAttribute("style", style);
-    const rightEditor = document.createElement("div");
-    rightEditor.setAttribute("style", style);
-    divRoot.appendChild(leftEditor);
-    divRoot.appendChild(rightEditor);
+    const table = document.createElement("table");
```

Comment:
The new layout looks good. But can't we define it in the static html file instead of creating it in JS?

Reply:
So true. Yes, I can do it.

Reply:
Done